src/content/decentralization/index.md

```diff
@@ -18,57 +18,144 @@ summaryPoints:
 
-Bonus: If you know anything at all about JavaScript or Python, you already have a huge head start.
+Ethereum is also *centralized* in a very important way — there is an agreed-upon set of rules (in the form of computer code) that every node should follow. This allows the system to operate predictably. However, it bears mentioning that these rules are upgraded and maintained in a decentralized way.
+
+### Smart contracts {#smart-contracts}
+
+Behind every dapp (decentralized app) on Ethereum, there is a smart contract. This is a program that runs on the Ethereum network for users to interact with.
+
+Smart contracts are an essential part of Ethereum's decentralized infrastructure. They can theoretically provide any service that a computer can. They can execute currency exchanges, retrieve and display information, make calculations, and so on.
+
+We say that Ethereum can solve "theoretically" any problem because there are some limits: namely the expensive digital real estate in the [Ethereum Virtual Machine](https://ethereum.org/en/glossary/#evm) and on the [blockchain](https://ethereum.org/en/glossary/#blockchain). 
+
+## Pros and cons of using Ethereum {#pros-cons}
+
+Let's say you're deciding where to send and store money. Would Ethereum be a good choice?
+
+### Pros {#pros-of-ethereum}
+
+- You have control over your funds.
+- It would take a complex, coordinated, international attack of disproportionate cost to stop the Ethereum network from operating.
+- You don't need permission to participate (no loan applications, for example). All you need is access to the network.
+- Ethereum does not associate your activities with your personal identity (because it doesn't know who you are).
+- Brute-force "identity theft" crimes can be extremely difficult because account credentials are cryptographically protected.
+- You can issue transactions instantly from anywhere with an internet connection.
+
+### Cons {#cons-of-ethereum}
+
+- Nobody but the recipient can help if you send your funds to the wrong address.
+- You are responsible for account security and privacy practices: for keeping your [private keys](https://ethereum.org/en/glossary/#private-key) secure and your personal information separate from your online activities.
+- You will probably have to learn new information and behaviors. Currently, even the most user-friendly apps can be technically challenging.
+- It is your responsibility to identify and comply with applicable laws.
+- It can be difficult to know which contracts you should trust.
+
+In the end, it's up to you to decide. However, know that there are a lot of smart people who are dedicating themselves to making this technology easier for you to use. If it's not the best choice for you now, check it out again in a few months.
+
+## Trust and consequences {#trust-in-decentralization}
```

Comment:
This section feels like it belongs above when we talk about why decentralized services are important